images/win/scripts/Installers/Install-DotnetSDK.ps1

```diff
@@ -108,6 +108,17 @@ function InstallAllValidSdks()
     }
 }
 
+function InstallTools()
+{
+    $dotnetToolset = (Get-ToolsetContent).dotnet
+    $dotnetTools = $dotnetToolset.tools
+
+    ForEach ($dotnetTool in $dotnetTools)
```

Comment:
FoEach -> foreach

Reply:
what's wrong here? can't see...

Reply:
I see we use ForEach notation in this script. Nothing wrong.